redirects.yaml

```diff
@@ -982,7 +983,6 @@ node/2316/?: "https://insights.ubuntu.com/"
 occs/?: "http://partners.ubuntu.com/ibm"
 ods/?: "https://insights.ubuntu.com/2014/11/03/extreme-openstack/?utm_source=ubunteu&utm_medium=url_shortner&utm_term=ods2014paris&utm_campaign=shortner"
 oil/?: "http://partners.ubuntu.com/programmes/openstack"
-openstack/?: "/cloud/openstack"
```

Comment:
I imagine we would want to add this in reverse, to redirect `/cloud/openstack` to `/openstack`

Reply:
This exists on line 73

Reply:
Ah, it isn't with the other alphabetical lines. 🤔